### PR TITLE
{Release} Upgrade to Azure CLI 2.57.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,113 @@
...
+* `az functionapp create`: Use app insights connection string instead of instrumentation key (#27803)
+* `az webapp create`: Add new parameter `--acr-use-identity` to configure pull image from ACR using MSI when creating a container web app (#28209)
+* `az webapp up`: Add `--track-status` arg to use deploymentstatus API (#28235)
+* `az webapp deployment source config-zip`: Add `--track-status` arg to use deploymentstatus API (#28235)
+* `az functionapp create`: Only allow apps with functions version 4 (#28128)
+
+**Backup**
+
+* Add support for disk restore in edge-zone backups (#28150)
+* Support for Taiwan North and Taiwan North-West Cross region restore (#28150)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:26]:
```suggestion
* `az backup restore restore-disks`: Add support for disk restore in edge-zone backups (#28150)
* `az backup restore restore-disks`: Support for Taiwan North and Taiwan North-West Cross region restore (#28150)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,113 @@
...
+* `az billing invoice section`: Add new command `initiate-transfer` to support intiating transfer (#28214)
+* `az billing profile`: Add new command `reservation list ` to support managing reservation (#28214)
+* `az billing`: Add new command group `transfer` to support managing transfer (#28214)
+
+**Compute**
+
+* `az image builder error-handler`: Add group to manage error handling options upon a build failure (#28106)
+* `az sshkey create`: Add parameter `--encryption-type` to specify the encryption type of SSH keys to be generated (#28143)
+* `az vm monitor log show`: Deprecate `azure-loganalytics` and apply `azure-monitor-query` (#28199)
+* `az vm create` and `az vmss create`: Support Trusted Launch as default deployment option (#28222)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:42]:
```suggestion
* `az vm/vmss create`: Support Trusted Launch as default deployment option (#28222)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,113 @@
...
+
+**Key Vault**
+
+* `az keyvault create`: Add warning for upcoming breaking change that `--enable-rbac-authorization` will default to `true` (#28178)
+* `az keyvault secret set-attributes`: Fix 'Datetime with no tzinfo will be considered UTC' warning (#28138)
+
+**Monitor**
+
+* `az monitor metrics`: Add `list-sub` and `list-sub-definition` (#28201)
+
```
Comment (AllyW) [inline src/azure-cli/HISTORY.rst:61]:
```suggestion
`az monitor metrics list`: Fix `--top` help message (#28114)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,113 @@
...
+* `az postgres flexible-server private-endpoint-connection`: Add support for private endpoint connection commands like list, show, approve, reject, delete for PostgreSQL flexible server (#28142)
+* `az postgres flexible-server private-link-resource`: Add support for private link resource commands like list, show for PostgreSQL flexible server (#28142)
+* `az postgres flexible-server replica stop-replication`: Stop replication to a read replica and make it a read/write server. This command is deprecated. Use `az postgres flexible-server replica promote` instead. (#28189)
+
+**Redis**
+
+* Add support for flush operation (#27599)
+* Add support for Microsoft Entra Authentication (#27599)
+* Add support for update channels (#27599)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:90]:
```suggestion
* `az redis flush`: Add support for flush operation (#27599)
* `az redis create`: Add support for Microsoft Entra Authentication (#27599)
* `az redis access-policy/access-policy-assignment`: New command groups to manage access policy (#27599)
```

File: src/azure-cli/HISTORY.rst (hunk 3 above)
Comment (AllyW) [inline src/azure-cli/HISTORY.rst:61]:
```suggestion
* `az monitor metrics list`: Fix `--top` help message (#28114)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,113 @@
...
+
+**Key Vault**
+
+* `az keyvault create`: Add warning for upcoming breaking change that `--enable-rbac-authorization` will default to `true` (#28178)
+* `az keyvault secret set-attributes`: Fix 'Datetime with no tzinfo will be considered UTC' warning (#28138)
+
+**Monitor**
+
+* `az monitor metrics`: Add `list-sub` and `list-sub-definition` (#28201)
+* `az monitor metrics list`: Fix `--top` help message (#28114)
```
Comment (MoChilia) [inline src/azure-cli/HISTORY.rst:61]:
```suggestion
* `az monitor metrics list`: Fix `--top` help message (#28114)

```

**Review by evelyn-ys — APPROVED** [review]

**Review by MoChilia — APPROVED** [review]

**Review by AllyW — DISMISSED** [review]

**Review by bebound — APPROVED** [review]

**Review by wangzelin007 — DISMISSED** [review]

File: src/azure-cli-core/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+2.57.0
+++++++
+* `aaz`: Handle null value when deserialize output (#28253)
+* Warn customers when there're credentials in cli output if this feature has been turned on through `az config set clients.show_secrets_warning=True` (#27929)
```
Comment (evelyn-ys) [inline src/azure-cli-core/HISTORY.rst:9]:
```suggestion
* Add warning for customers when there're credentials in cli output if this feature has been turned on through `az config set clients.show_secrets_warning=True` (#27929)
```

**Review by evelyn-ys — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

**Review by MoChilia — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by AllyW — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Pull request merged**